pxe-server/README.md

```diff
@@ -0,0 +1,38 @@
+<!-- SPDX-FileCopyrightText: (C) 2025 Intel Corporation -->
```

Comment:
this README has to adhere to the template. Also please add elements to the top level readme.md as well.